runtime/litentry/src/transaction_payment.rs

```diff
@@ -31,16 +31,8 @@ where
 	<R as frame_system::Config>::Event: From<pallet_balances::Event<R>>,
 {
 	fn on_nonzero_unbalanced(amount: NegativeImbalance<R>) {
-		let numeric_amount = amount.peek();
 		let author = <pallet_authorship::Pallet<R>>::author();
-		<pallet_balances::Pallet<R>>::resolve_creating(
-			&<pallet_authorship::Pallet<R>>::author(),
-			amount,
-		);
-		<frame_system::Pallet<R>>::deposit_event(pallet_balances::Event::Deposit {
-			who: author,
-			amount: numeric_amount,
-		});
```

Comment:
we don't need any event here, right? As cumulus seems to have one: https://github.com/paritytech/cumulus/blob/master/polkadot-parachains/parachains-common/src/impls.rs#L50-L52

Reply:
Yes. I think they are wrong.
https://github.com/paritytech/substrate/blob/master/frame/support/src/traits/tokens/currency.rs#L158
https://github.com/paritytech/substrate/blob/master/frame/balances/src/lib.rs#L1647
Which obviously generating two events of Deposit.

Reply:
Let me try PR them, hahaha

Reply:
> Yes. I think they are wrong. https://github.com/paritytech/substrate/blob/master/frame/support/src/traits/tokens/currency.rs#L158 https://github.com/paritytech/substrate/blob/master/frame/balances/src/lib.rs#L1647 Which obviously generating two events of Deposit.

Yea maybe want to trigger such event in any case, but using the Curreny Event should be cleaner.